backend/accounting-service/src/jvmMain/kotlin/services/projects/ProjectService.kt

```diff
@@ -84,6 +84,10 @@ class ProjectService(
         parent: String?,
         principalInvestigatorOverride: String?
     ): String {
+        if (title.trim().length != title.length) {
+            throw RPCException("Project names cannot start or end with whitespace.", HttpStatusCode.BadRequest)
+        }
```

Comment:
Should this be in an init-function for the data-class instead?

Reply:
Should be fine here.